mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSerDe.java

```diff
@@ -137,15 +137,21 @@ private static Schema hiveSchemaOrThrow(Properties serDeProperties, Exception pr
     // Read the configuration parameters
     String columnNames = serDeProperties.getProperty(serdeConstants.LIST_COLUMNS);
     String columnTypes = serDeProperties.getProperty(serdeConstants.LIST_COLUMN_TYPES);
+    // No constant for column comments and column comments delimiter.
```

Comment:
not sure I follow this comment..

Reply:
What I meant here, that there is no constant definition among the serdeConstants, therefore I had to hardcode the column comment ("columns.comments") and column delimiter (Character.MIN_VALUE) property names.